src/snowflake/snowpark/modin/pandas/base.py

```diff
@@ -1045,21 +1045,25 @@ def error_raiser(msg, exception):
         )
         return query_compiler
 
-    @base_not_implemented()
     def asfreq(
-        self, freq, method=None, how=None, normalize=False, fill_value=None
+        self,
+        freq: str,
+        method: FillnaOptions | None = None,
+        how: str | None = None,
+        normalize: bool = False,
+        fill_value: Scalar = None,
     ):  # noqa: PR01, RT01, D200
         """
         Convert time series to specified frequency.
         """
-        # TODO: SNOW-1119855: Modin upgrade - modin.pandas.base.BasePandasDataset
```

Comment:
SNOW-1119855 is not resolved yet. Are we sure we want to remove this TODO?

Reply:
This TODO/ticket is in several places, but the frontend method now matches Modin so we do want to remove the TODO here.